src/Files.App/ViewModels/Properties/Items/FolderProperties.cs

```diff
@@ -107,10 +108,13 @@ public async override Task GetSpecialPropertiesAsync()
 			{
 				ViewModel.ItemCreatedTimestampReal = storageFolder.DateCreated;
 				if (storageFolder.Properties is not null)
-				{
 					GetOtherPropertiesAsync(storageFolder.Properties);
-				}
-				GetFolderSizeAsync(storageFolder.Path, TokenSource.Token);
+
+				// Only load the size for items on the device
```

Comment:
Is it necessary? Calcuating folder size is not causing files to download for me.

Reply:
Are there files in the folder?

Reply:
Yeah
<img width="756" alt="image" src="https://github.com/files-community/Files/assets/9673091/73cee576-7080-461a-82a2-167483d38b57">

Reply:
What happens if you "Free up space" and then open properties?

Reply:
It shows a smaller size on disk
<img width="267" alt="image" src="https://github.com/files-community/Files/assets/9673091/517f2acb-6652-4b71-b3f7-59836199d5ac">

Reply:
Can you place a breakpoint on line 111 and let me know the `SyncStatus`?

Reply:
Aww right it's still OfflinePartial even if I did "clear space", one sec.

Reply:
Yup stays OfflinePartial but all files inside are online only.